packages/botonic-cli/src/utils.ts

```diff
@@ -54,9 +54,13 @@ if (isAnalyticsEnabled()) {
   analytics = new Analytics('YD0jpJHNGW12uhLNbgB4wbdTRQ4Cy1Zu')
 }
 
-function getBotonicCliVersion(): string {
+function isWindows() {
+  return process.platform === 'win32'
```

Comment:
Small observation, I know it's the same, but I would use here `os.platform()` instead of `process.platform` just to keep using the same methods everywhere (we're using `os.platform()` later in `getSystemInformation()`)

Reply:
nice observation, applied! 👍